public/app/plugins/datasource/prometheus/datasource.ts

```diff
@@ -322,7 +330,7 @@ export class PrometheusDatasource implements DataSourceApi<PromQuery> {
 
   getRangeScopedVars() {
     const range = this.timeSrv.timeRange();
-    const msRange = range.to.diff(range.from);
+    const msRange = (range.to as any).diff(range.from); // CHECK!!!!
```

Comment:
@torkelo -- I think this is based on the recent wrapper to time values... maybe we need to add diff?

Reply:
yes, that must be it. because diff does exist on moment

Reply:
I added diff to TimeValue